api-update/Jenkinsfile-test

```diff
@@ -0,0 +1,67 @@
+def versionTag = "v${env.BUILD_NUMBER}"
+
+pipeline {
+    agent {
+        label "master"
+    }
+    parameters {
+        choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are building the front end for")
+    }
+    stages {
+        stage("Build") {
+            agent {
+                ecs {
+                    inheritFrom "transfer-frontend"
+                }
+            }
+            steps {
+                dir("api-update") {
+                    sh "sbt test antivirus/assembly"
+                    sh "mv antivirus/target/scala-2.13/antivirus-assembly-*.jar antivirus/target/scala-2.13/antivirus.jar"
+                    stash includes: "antivirus/target/scala-2.13/antivirus.jar", name: "antivirus-jar"
+                }
+            }
+        }
+        stage('Post-build') {
+            agent {
+                ecs {
+                    inheritFrom "aws"
+                    taskrole "arn:aws:iam::${env.MANAGEMENT_ACCOUNT}:role/TDRJenkinsNodeLambdaRole${params.STAGE.capitalize()}"
+                }
+            }
+            when {
+              expression { env.BRANCH_NAME == "master"}
+            }
+
+            stages {
+                stage('Deploy to integration') {
+                    steps {
+                        script {
+                            unstash "antivirus-jar"
+                            sh "cp antivirus/target/scala-2.13/antivirus.jar /"
```

Comment:
It's copying to the root of the build container which is destroyed once the job is complete. There is a ticket somewhere to stop running all containers as root and this will get picked up with that but as it's getting destroyed, it's ok for now.

Reply:
Do you need to copy it elsewhere? Could you just refer to `antivirus/target/scala-2.13/antivirus.jar` when you run the python script on line 46?

Reply:
The python scripts are in the root directory and the unstashed files aren't so I'd have to get the full path to the file. I could but this is fine and as I said, it's destroyed after.